packages/debugger/src/panels/variables/tree.tsx

```diff
@@ -452,57 +456,47 @@ const VariableComponent = (props: IVariableComponentProps): JSX.Element => {
   };
 
   return (
-    <li
-      onClick={(e): Promise<void> => onVariableClicked(e)}
-      onMouseDown={e => {
-        e.stopPropagation();
-        onSelection(variable);
-      }}
-      onMouseOver={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
-        if (onHoverChanged) {
-          onHoverChanged({ target: event.currentTarget, variable });
-          event.stopPropagation();
-        }
-      }}
-      onMouseLeave={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
-        if (onHoverChanged) {
-          onHoverChanged({
-            target: event.relatedTarget as EventTarget & HTMLElement,
-            variable: null
-          });
-          event.stopPropagation();
-        }
-      }}
-    >
-      <span
-        className={
-          'jp-DebuggerVariables-collapser' +
-          (expanded ? ' jp-mod-expanded' : '')
-        }
+      <TreeItem
+        expanded={expanded}
+        onClick={(e): Promise<void> => onVariableClicked(e)}
+        onMouseDown={e => {
+          e.stopPropagation();
+          onSelection(variable);
+        }}
+        onMouseOver={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
+          if (onHoverChanged) {
+            onHoverChanged({ target: event.currentTarget, variable });
+            event.stopPropagation();
+          }
+        }}
+        onMouseLeave={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
+          if (onHoverChanged) {
+            onHoverChanged({
+              target: event.relatedTarget as EventTarget & HTMLElement,
+              variable: null
+            });
+            event.stopPropagation();
+          }
+        }}
       >
-        {
-          // note: using React.cloneElement due to high typestyle cost
-          expandable ? React.cloneElement(collapserIcon) : null
-        }
-      </span>
-      <span className="jp-DebuggerVariables-name">{variable.name}</span>
-      <span className="jp-DebuggerVariables-detail">
-        {_prepareDetail(variable)}
-      </span>
-      {expanded && variables && (
-        <VariablesBranch
-          key={variable.name}
-          commands={commands}
-          data={variables}
-          service={service}
-          filter={filter}
-          translator={translator}
-          handleSelectVariable={onSelect}
-          onHoverChanged={onHoverChanged}
-          collapserIcon={collapserIcon}
-        />
-      )}
-    </li>
+        <span className="jp-DebuggerVariables-name">{variable.name}</span>
+        <span className="jp-DebuggerVariables-detail">
+          {_prepareDetail(variable)}
+        </span>
+        {expanded && variables && (
```

Comment:
For the debugger variables view, I'm seeing an issue with the collapse button not being displayed:

![Peek 2024-06-19 10-03](https://github.com/jupyterlab/jupyterlab/assets/8435071/147fb2b5-4a77-4d88-bf01-df0cd83278a1)

I think part of the issue is due to the `expanded` condition here that should not be used any longer:

```suggestion
        {variables && (
```